views/BeerView.js

```diff
@@ -0,0 +1,61 @@
+var BeerView = Backbone.View.extend({
+  className: 'beer',
+  
+  events: {
+    'click .remove-beer': 'removeBeer',
+    'dblclick .view': 'editBeer',
+    // 'blur .edit': 'saveBeer',
+    'keypress .editName': 'updateOnEnter'
+  },
+
+  template: Handlebars.compile($('#beer-template').html()),
+
+  initialize: function () {
+    this.listenTo(this.model, 'destroy', this.remove),
+    this.listenTo(this.model, 'change', this.render)
+  },
+
+  render: function() {
+    this.$input = this.$('.edit');
+    this.$nameInput = this.$('.editName');
+    this.$el.html(this.template(this.model.toJSON()));
+
+    return this;
+  },
+
+  removeBeer: function () {
+    this.model.destroy();
+  },
+
+  editBeer: function() {
+    $('.view').addClass('hidden');
```

Comment:
I know the todoMVC example had this kind of thing happening, but I'm not a big fan... by "this kind of thing" I mean toggling some CSS without first consulting the model (it violates our rules of data-flow). I would toggle some kind of attribute on the model like `edit_mode: false`, then have the view respond to a change on model's the `edit_mode` attribute.